filecoin-proofs/src/api/internal.rs

```diff
@@ -447,7 +485,7 @@ fn make_merkle_tree<T: Into<PathBuf> + AsRef<Path>>(
     let mut data = Vec::new();
     f_in.read_to_end(&mut data)?;
 
-    public_params(bytes).graph.merkle_tree(&data)
+    public_params(bytes, 1).graph.merkle_tree(&data)
```

Comment:
QUESTION: Why the hard-coded 1 here? `make_merkle_tree` is called during PoSt generation. I would expect that PoSt proof partitions would be plumbed in to `make_merkle_tree` instead of this hard-coded 1.

Reply:
I intended that as a later step. As we discussed, PoSt doesn't currently support multiple partitions so I defaulted to what it does — until such time as we need and plumb otherwise.